### DISCUSSION TITLE
如果能根据元数据设置目录名就更好了，谢谢！

### DISCUSSION THREAD

**Discussion by chinaeaa** [Cirn09/calibre-do-not-translate-my-path#20] [Ideas]
如果能根据元数据可控设置目录名就更好了……
并非是真的发issues，主要是对你表示感谢！
解决了一直困扰我的大问题，再次感谢！
好人一生平安！

**Comment by Cirn09** [comment]
功能和原版差异越大越难通过本项目的 patch 方式实现，特别是给原版加功能

真的需要的话不如去说服原版作者合入
